README.md

```diff
@@ -1,161 +1,279 @@
-# hello-microverse
-This is just basic min-project to must github flow 
+<a name="readme-top"></a>
 
- 
- ![murple_logo](https://user-images.githubusercontent.com/110924992/218853728-1a8f5726-cd6a-4613-85ed-38dd5132cd9f.png)
- 
- 
- 
- 
 
- 
-Microverse README Template
-📗 Table of Contents
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-📖 About the Project
+  <h3><b>Hello Microverse</b></h3>
 
-🛠 Built With
-html
-& css
+</div>
```

Comment:
- [ ] Kindly consider removing this microverse logo or add yours if you have any cause  this will make your repo personal.